### PR TITLE
Security - added message when not logged in

### PR DESCRIPTION
**Pull request by AndyC2513**
yes

### CODE REVIEW

File: frontend/src/Test.js [hunk 1]
```diff
@@ -62,6 +62,8 @@ function Test() {
         />
       </div>
 
+      {user ? null : <div>Please Log In</div>}
```
Comment (joshhwuu) [inline frontend/src/Test.js:65]: Is it possible to just reroute users to the login page?

Comment (AndyC2513) [reply frontend/src/Test.js:65]: Very interesting concept, I will look into it! :D

Comment (joshhwuu) [inline frontend/src/Test.js:65]: Is it possible to move this check to the top? No use checking null at the very end after checking everything else, instead we should send user straight to login page at the very beginning

**Comment by jeffreyzhai123** [comment]
please fix based on josh's suggestions.

**Comment by AndyC2513** [comment]
Fixed, now routes back to home page if not logged in

**Review by jeffreyzhai123 — APPROVED** [review]

**Pull request merged**